com.unity.shadergraph/Editor/GraphDeltaRegistry/Registry/GradientType.cs

```diff
@@ -76,10 +76,10 @@ public ShaderFoundry.ShaderFunction GetShaderFunction(NodeHandler node, ShaderFo
     /// </summary>
     internal class GradientNode : INodeDefinitionBuilder
     {
-        public RegistryKey GetRegistryKey() => new RegistryKey { Name = "GradientNode", Version = 1 };
+        public RegistryKey GetRegistryKey() => new RegistryKey { Name = "Gradient", Version = 1 };
         public RegistryFlags GetRegistryFlags() => RegistryFlags.Func;
 
-        public const string kInlineStatic = "InlineStatic";
+        public const string kInlineStatic = " ";
```

Comment:
@bencloward "GradientNode" and "InlineStatic" here are reference names in the generated code ("Gradient" is reserved and " " isn't a valid identifier). They are also used for display names right now, but we will be introducing another way to override those.